net6.0-android: Java.Interop conflict

# Summary
Referencing `Silk.NET.Windowing` 2.10.0.
Compiling result in following error:
```
1>CSC : error CS1705: Assembly 'Silk.NET.Windowing.Sdl' with identity 'Silk.NET.Windowing.Sdl, Version=2.10.0.0, Culture=neutral, PublicKeyToken=null' uses 'Java.Interop, Version=0.1.2.0, Culture=neutral, PublicKeyToken=84e04ff9cfb79065' which has a higher version than referenced assembly 'Java.Interop' with identity 'Java.Interop, Version=0.1.0.0, Culture=neutral, PublicKeyToken=84e04ff9cfb79065'
```

# Steps to reproduce
- Platform: Android
- Framework Version: `net6.0-android`
- API: OpenGL

Add following references:
```
    <PackageReference Include="Silk.NET.OpenGL" Version="2.10.0" />
    <PackageReference Include="Silk.NET.Windowing" Version="2.10.0" />
```

# Comments
If I use 2.9.0, it works fine.
I suspect 2.10.0 was compiled with RC3 which had a bug: https://github.com/xamarin/xamarin-android/issues/6438
Probably, recompiling 2.11.0 with VS2022 RTM might fix the issue.

--- a/build/props/common.props
+++ b/build/props/common.props
@@ -5,28 +5,16 @@
         <LangVersion>preview</LangVersion>
         <Authors>.NET Foundation and Contributors</Authors>
         <PackageReleaseNotes>
-            Silk.NET November 2021 Update
+            Silk.NET .NET Conf 2021 Update
             
-            - Added support for BigInteger and Complex in the Silk.NET.Maths.Scalar APIs. (thanks @WhiteBlackGoose)
-            - Added helper constructors for maths types (Rectangle, Box2D, Box3D, etc). (thanks @nathan-alden-sr)
-            - Added bitwise APIs to Silk.NET.Maths.Scalar. (thanks @WhiteBlackGoose)
-            - Added support for configuring OpenGL multisamping.
-            - Updated to Vulkan 1.2.197.
-            - Updated to latest OpenCL specifications.
-            - Updated to latest OpenGL specifications.
-            - Updated DirectX and Win32 bindings to Windows 11 SDK.
-            - Removed redundant references with no DLLs (i.e. BCL/legacy packages not needed on newer platforms, thanks @nathan-alden-sr @WhiteBlackGoose)
-            - Removed problematic implicit-in OpenGL overloads (fixes the infamous VertexAttribPointer bug)
-            - Fixed incorrect character encoding of keyboard (text) input on the SDL backend. (thanks @roeyskoe)
-            - Fixed native libraries not being resolved despite being present in the "runtimes" folder.
-            - Miscellaneous documentation fixes and improvements.
+            - Fixes issues with 2.10 (compiled against .NET 6 RC1) not working correctly in projects using the .NET 6 Release SDK.
         </PackageReleaseNotes>
         <PackageTags Condition="'$(PackageTags)' == ''">OpenCL;OpenGL;OpenAL;OpenGLES;GLES;Vulkan;Assimp;DirectX;GLFW;SDL;Windowing;Input;Gamepad;Joystick;Keyboard;Mouse;SilkTouch;Source;Generator;C#;F#;.NET;DotNet;Mono;Vector;Math;Maths;Numerics;Game;Graphics;Compute;Audio;Sound;Engine;Silk;Silk.NET;Slim.NET;ElgarTK;GPU;Sharp;Science;Scientific;Visualization;Visual;Audiovisual;Windows;macOS;Linux;Android;Bindings;OSX;Wrapper;Native</PackageTags>
         <GenerateDocumentationFile>true</GenerateDocumentationFile>
         <PackageOutputPath>$(MSBuildThisFileDirectory)/../output_packages</PackageOutputPath>
         <RepositoryUrl>https://github.com/dotnet/Silk.NET</RepositoryUrl>
         <RepositoryType>Git</RepositoryType>
-        <VersionPrefix>2.10.0</VersionPrefix>
+        <VersionPrefix>2.10.1</VersionPrefix>
         <VersionSuffix Condition="'$(VersionSuffix)' == ''"></VersionSuffix>
         <Description Condition="'$(Description)' == ''">
             Silk.NET is a high-speed, advanced library, providing bindings to popular low-level APIs such as OpenGL, OpenCL, OpenAL, OpenXR, GLFW, SDL, Vulkan, Assimp, and DirectX.

--- a/src/Core/Silk.NET.BuildTools/Silk.NET.BuildTools.csproj
+++ b/src/Core/Silk.NET.BuildTools/Silk.NET.BuildTools.csproj
@@ -15,7 +15,7 @@
       <PackageReference Include="CommandLineParser" Version="2.8.0" />
       <PackageReference Include="HtmlAgilityPack" Version="1.11.24" />
       <PackageReference Include="Humanizer.Core" Version="2.8.26" />
-      <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.8.0-3.final" />
+      <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.11.0" />
       <PackageReference Include="Microsoft.Extensions.FileSystemGlobbing" Version="3.1.8" />
       <PackageReference Include="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="2.3.2262-g94fae01e" />
       <PackageReference Include="morelinq" Version="3.3.2" />

